src/engraving/draw/pen.h

```diff
@@ -39,8 +39,8 @@ class Pen
     {
     }
 
-    Pen(const QColor& color = Qt::black, double width = 1, PenStyle s = PenStyle::SolidLine, PenCapStyle c = PenCapStyle::SquareCap,
-        PenJoinStyle j = PenJoinStyle::BevelJoin)
+    Pen(const mu::draw::Color& color = mu::draw::black, double width = 1, PenStyle s = PenStyle::SolidLine,
```

Comment:
There is no need to specify the namespace mu::draw::, they are in the same namespace